charts/tempo-distributed/values.yaml

```diff
@@ -705,17 +705,38 @@ config: |
         kvstore:
           store: "memberlist"
 
-  {{- if .Values.minio.enabled }}
+  auth:
+    type: enterprise
```

Comment:
We shouldn't have this hard-coded. AFAIK, we want to support GET and OSS Tempo in the same chart.

Reply:
Good catch, I'd meant to change that!

Reply:
We're in the block for `Values.enterprise.enabled`, so I think we still keep our desires of OSS and GET together.